From 5042253f63aa70a781b1dd56d42852eba9020539 Mon Sep 17 00:00:00 2001
From: Kexy Biscuit <kexybiscuit@aosc.io>
Date: Fri, 26 Jul 2024 10:52:16 +0800
Subject: [PATCH 25/31] IWYU: missing include for usage of std::exchange in
 lock_impl.h

Bug: 41455655
Change-Id: If324eb8eece3dc710e6bcc032cc0587f45dd876b
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/5616412
Commit-Queue: Lei Zhang <thestig@chromium.org>
Reviewed-by: Lei Zhang <thestig@chromium.org>
Reviewed-by: Bartek Nowierski <bartekn@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1313273}

Co-authored-by: Jose Dapena Paz <jdapena@igalia.com>
---
 base/synchronization/lock_impl.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/base/synchronization/lock_impl.h b/base/synchronization/lock_impl.h
index 6289b0a9a8..af4f268536 100644
--- a/base/synchronization/lock_impl.h
+++ b/base/synchronization/lock_impl.h
@@ -5,6 +5,8 @@
 #ifndef BASE_SYNCHRONIZATION_LOCK_IMPL_H_
 #define BASE_SYNCHRONIZATION_LOCK_IMPL_H_
 
+#include <utility>
+
 #include "base/base_export.h"
 #include "base/check.h"
 #include "base/dcheck_is_on.h"
-- 
2.45.2

